lib/modules/csv_exporter.rb

```diff
@@ -11,6 +11,8 @@ def self.export_results(from_date, to_date)
     CSV.open(filepath, "wb") do |csv|
       csv << self.headers(latest)
 
+      from_date = from_date.empty? || !self.date_valid?(from_date) ? "2016-01-01" : from_date
```

Comment:
Is there a better way to do this? If we don't have a date we don't even need the where

Reply:
Yes, that's a good point. I guess if both the `from_date` and the `to_date` are empty then I can totally miss out the `where` completely.